test/snippet/range/view/persist.cpp

```diff
@@ -0,0 +1,31 @@
+#include <seqan3/alphabet/nucleotide/dna4.hpp>
+#include <seqan3/range/view/persist.hpp>
+#include <seqan3/range/view/to_char.hpp>
+
+using namespace seqan3;
+using namespace seqan3::literal;
+
+int main()
+{
+//! [usage]
```

Comment:
include the `main` and the `#includes` into the example. I think larger examples should be copy and pastable.

Reply:
lets discuss this with the other snippets